cranelift/codegen/src/isa/x64/inst/emit_tests.rs

```diff
@@ -2688,6 +2688,13 @@ fn test_x64_emit() {
         "440F56F9",
         "orps    %xmm1, %xmm15",
     ));
+
+    insns.push((
+        Inst::xmm_mov_r_m(SseOpcode::Movd, xmm0, Amode::imm_reg(321, rbx)),
+        "660F7E8341010000",
+        "movd    %xmm0, 321(%rbx)",
```

Comment:
Can you add one test that uses one of the high registers requiring the REX prefix, please?

Reply:
Right ... I've done this with a new test for Movss. Thanks for pushing on the movss 👍, not sure what I was overlooking there.